Form-Controls/index.html

```diff
@@ -1,27 +1,54 @@
-<!DOCTYPE html>
-<html lang="en">
-  <head>
-    <meta charset="utf-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <title>My form exercise</title>
-    <meta name="description" content="" />
-    <meta name="viewport" content="width=device-width, initial-scale=1" />
-  </head>
-  <body>
-    <header>
-      <h1>Product Pick</h1>
-    </header>
-    <main>
-      <form>
-        <!-- write your html here-->
-        <!-- 
-          try writing out the requirements first as comments
-        this will also help you fill in your PR message later-->
-      </form>
-    </main>
-    <footer>
-      <!-- change to your name-->
-      <h2>By HOMEWORK SOLUTION</h2>
-    </footer>
-  </body>
-</html>
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title>My form exercise</title>
+    <meta name="description" content="" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+  </head>
+  <body>
+    <header>
+      <h1>Customers'T-shirt Pick</h1>
+    </header>
+    <main>
+      <form>
+      <div>
+        <label for="username"> Enter username:</label>
+        <input type= "text" id="username" name="username" placeholder= "Username" pattern="[A-Za-z]{34-10"} required  autocomplete="off"/><br><br>   
+      </div>
+      <div>
+        <label for="email">Enter your email:</label>
+        <input type="email" id="email" name="email" placeholder="Email" required autocomplete="off"/><br><br>
+      </div>
+      <div>
+        <label for="T-shirtcolour">Choose a colour:</label>
+        <select id="T-shirtcolour" name="T-shirtcolour">

```

Comment:
We would still want the `select` element to have the `required` attribute here - that will enforce that a user has to select one of the colours, rather than leaving "Colour" selected. Do try the form out - having a quick runthrough of the interface is a good part of the testing process to check whether you are done :)